maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationServiceTest.java

```diff
@@ -119,13 +122,13 @@ void givenARequestWithC3Enabled_whenCreateIsInvoked_thenApplicationDTOIsUpdatedW
     void givenARequestWithC3NotEnabled_whenCreateIsInvoked_thenCalculationContributionIsNotCalled() {
 
         WorkflowRequest workflowRequest = MeansAssessmentDataBuilder.buildWorkFlowRequest();
-
+        RepOrderDTO repOrderDTO = TestModelDataBuilder.getTestRepOrderDTO(workflowRequest.getApplicationDTO());
```

Comment:
Can we could consolidate some of the repeated stub definitions into a `@BeforeAll` or other dedicated method to cut down on some of the duplication please.

Reply:
Done